benchmark.js

```diff
@@ -1,5 +1,5 @@
-const Benchmark = require('benchmark');
-const SuperJSON = require('./dist/').default;
+import Benchmark from 'benchmark';
+import SuperJSON from './dist/index.js';
```

Comment:
The reason to use `require` here was that it enabled running `node benchmark.js`, which is quite helpful in dev. Is this change neccessary?

Reply:
Running `node benchmark.js` fails after adding `"type": "module"` if using `require`.

Reply:
Added a script to ensure the benchmark can be run in Node 10 as well, by enabling modules with the experimental flag. You can now run `yarn benchmark`.